t/spamd_prefork_stress_3.t

```diff
@@ -2,19 +2,17 @@
 
 use lib '.'; use lib 't';
 use SATest; sa_t_init("spamd_prefork_stress_3");
-use Test;
 
-our $RUN_THIS_TEST;
-BEGIN {
-  $RUN_THIS_TEST = conf_bool('run_spamd_prefork_stress_test') &&
-                        conf_bool('run_long_tests');
-  plan tests => ($SKIP_SPAMD_TESTS || !$RUN_THIS_TEST ? 0 : 291)
+use Test::More;
+ BEGIN {
```

Comment:
is the BEGIN block required ? regarding the exit which were used to be there at run time pretty sure it s not required